ACE/debian/libace-VERSION.docs

```diff
@@ -1,8 +1,8 @@
-README
-VERSION.txt
-AUTHORS
+usr/share/ace/README
```

Comment:
Will these explicit paths not break things, not sure how this works for example on opensuse build service

Reply:
without the path the files are installed from the source folder and not from the `debian/tmp` where the files to be installed are kept. And so the debhelper thinks these files are not installed. With the paths mentioned debhelper looks for them in the debian/tmp and installs from there. I am not sure how `opensuse build service` works but confused why a Debian specific change in a Debian specific file will break `opensuse build service`.

Reply:
Ok, so it should keep working

Reply:
@jwillemsen yes. Debian has a tool called `debdiff` which can show the difference between deb packages and I verified before uploading that files were not missing from the binary packages.